conans/client/conf/__init__.py

```diff
@@ -12,8 +12,18 @@
 
 MIN_SERVER_COMPATIBLE_VERSION = '0.12.0'
 
-default_settings_yml = """os: [Windows, Linux, Macos, Android, iOS, FreeBSD, SunOS]
-arch: [x86, x86_64, ppc64le, ppc64, armv6, armv7, armv7hf, armv8, sparc, sparcv9]
+default_settings_yml = """
+os:
+    Windows:
+    Linux:
+    Macos:
+    Android:
+        api_level: ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "23", "24", "25"]
```

Comment:
Maybe ``api_level`` = ANY? So it is simpler and extensible. And being so simple, there is low risk of misspelling.